packages/utils/src/requestdata.ts

```diff
@@ -35,6 +34,7 @@ export type AddRequestDataToEventOptions = {
   include?: {
     ip?: boolean;
     request?: boolean | Array<(typeof DEFAULT_REQUEST_INCLUDES)[number]>;
+    /** @deprecated This option will be removed in v9. It does not do anything anymore, the `transcation` is set in other places. */
```

Comment:
but the request is quite different from this, isn't it? Opting out of the request there will not affect the transaction being set on the transaction event - though to be honest, it probably did not actually affect it even before, because that was set anyhow by other code already 😅